src/Google.Api.Gax/CallTiming.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    public enum CallTimingType
+    {
+        /// <summary>
+        /// Call uses retry, specified with a <see cref="RetrySettings"/>.
+        /// </summary>
+        Retry,
+
+        /// <summary>
+        /// Call does not use retry; call expiration is specified with an <see cref="Google.Api.Gax.Expiration"/> .
+        /// </summary>
+        Expiration,
+    }
+
+    public sealed class CallTiming
```

Comment:
The name "CallTiming" implies to me that it gives control around when the call is made. I have been thinking about what to name the combined concept of retry + timeout settings, and my current thought is "failure handling settings". None of these settings should cause anything to happen if an RPC succeeds and returns in the expected timeframe.